flutter_bird_skins/contracts/FlutterBirdSkins.sol

```diff
@@ -59,38 +62,23 @@ contract FlutterBirdSkins is ERC721Enumerable {
         return _unmintedTokes;
     }
 
-    function _baseURI() internal view virtual override returns (string memory) {
-        return "ipfs://bafybeiexvqt3uabqzmhquokzyl7gcdxyowz2hf2hdbbifkkyx3waghezqe/";
-    }
-
-    function tokenURI(uint256 tokenId) public view virtual override returns (string memory) {
-        require(_exists(tokenId), "ERC721URIStorage: URI query for nonexistent token");
-
-        string memory _tokenURI = _tokenURIs[tokenId];
-        string memory base = _baseURI();
-
-        // If there is no base URI, return the token URI.
-        if (bytes(base).length == 0) {
-            return _tokenURI;
+    function appendUri(uint256 tokenId, bytes[] calldata values) public onlyOwner {
```

Comment:
Can there be multiple URIs for one token?

Reply:
No, one token is assigned to one URI.